<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Date</class>
 <widget class="QDialog" name="Date">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>296</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectDate">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>XXXX-XX-XX</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumTime">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="displayFormat">
    <string>hh:mm:ss</string>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mulai jam</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectTime">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>XXXX-XX-XX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sampai jam</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumTime">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="displayFormat">
    <string>hh:mm:ss</string>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="selectTime_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>  -</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectTimeUntil">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>XXXX-XX-XX</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PLTMH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLTS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LOAD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INVERTER</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pilih data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
